addons/account_update_tax_tags/wizard/account_update_tax_tags_wizard.py

```diff
@@ -0,0 +1,169 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from datetime import timedelta
+from odoo import models, fields, api
+
+
+class AccountUpdateTaxTagsWizard(models.TransientModel):
+    _name = 'account.update.tax.tags.wizard'
+    _description = 'Update Tax Tags Wizard'
+
+    company_id = fields.Many2one(comodel_name='res.company', required=True, readonly=True, default=lambda self: self.env.company)
+    from_date = fields.Date(
+        string='Starting from',
+        help='Date from which journal entries will be updated.',
+        compute='_compute_from_date',
+        store=True,
+        readonly=False,
+        required=True,
+    )
+
+    # ==== Compute methods ====
+    @api.depends('company_id')
+    def _compute_from_date(self):
+        for wizard in self:
+            tax_lock_date = self.company_id.tax_lock_date
+            wizard.from_date = tax_lock_date + timedelta(days=1) if tax_lock_date else fields.Date.context_today(self)
+
+    # ==== Business methods ====
+    def _modify_tag_to_aml_relation(self, company_id, from_date, excluded_move_ids=None, accept_entry_type_move=True):
```

Comment:
Just for the support context, it was introduced by ALT to avoid updating moves that would be "touchy" to update automatically with a much more basic version of the SQL script.